app/controllers/lifestyle_footprints_controller.rb

```diff
@@ -29,8 +31,22 @@ def create
     end
 
     @country_average = LifestyleFootprintAverage.find_by_country(@footprint.country)
-    render json: {footprint: @footprint, country_average: @country_average}
-    # to redirect to result page, use below instead of 31 and 32
+
+    @footprint_tonnes = @footprint&.total
+    number_of_people = params[:membership] == 'multi' && params[:people].present? ? params[:people].to_i : 1
+    @plan = Subscriptions::Plan.for_footprint(@footprint_tonnes * number_of_people, current_region.currency)
+    
+    p "FPFPFPFPFP TONNES"
+    p @footprint_tonnes
+    p "NUM PPL"
+    p number_of_people
+    p "PLAAAAAAAAAN"
+    p @plan
+    p "DAPWDMAWIDJIAWDJIOAW"
```

Comment:
Hahahah är detta bra för debugging reasons, isåfall kan det få vara kvar!

Reply:
Haha ska bort, tack!